packages/libdbus/Cargo.toml

```diff
@@ -12,12 +12,12 @@ path = "../packages.rs"
 releases-url = "https://dbus.freedesktop.org/releases/dbus"
 
 [[package.metadata.build-package.external-files]]
-url = "https://dbus.freedesktop.org/releases/dbus/dbus-1.15.12.tar.xz"
-sha512 = "9af8ee28d61e1794eacd8cdecd6a68fa32ab5ea229df32f7112ed67f1aa8e57dd9a595ceaf42621891b50074e6f5a0cdd8c76f7d5882633446d0b81c40434cc0"
+url = "https://dbus.freedesktop.org/releases/dbus/dbus-1.16.2.tar.xz"
```

Comment:
dbus also uses the "stable releases use even numbers" versioning scheme, so let's add a comment for the releases url here too.